src/Pages/login/login.jsx

```diff
@@ -64,26 +64,30 @@ export default () => {
           </div>
 
           <div className="w-full">
-            <input
+            <InputFiled
               type="password"
-              placeholder="Password"
+              placeHolder="Password"
+              value={password}
               onBlur={() => {
                 setPasswordActivated(true);
-
                 checkPassword(password);
               }}
-              className="border p-1 rounded-md border-Gray drop-shadow w-full"
-              value={password}
-              onChange={(e) => setPassword(e.target.value)}
+              Onchange={(e) => setPassword(e.target.value)}
             />
-
             {isPasswordActivated && (
               <div>
                 <p className="text-red text-xs">{passwordAlert}</p>
               </div>
             )}
           </div>
         </div>
+
+        {isInputsValid === false && (
```

Comment:
```suggestion
        {!isInputsValid && (
```
This is a shorthand version of your check

Reply:
Fixed 8b7142b3bb568f0c1e361ef10bfeae717284050f